config/environments/development.rb

```diff
@@ -45,20 +45,15 @@
   # Highlight code that triggered database queries in logs.
   config.active_record.verbose_query_logs = true
 
+  # Highlight code that enqueued background job in logs.
+  config.active_job.verbose_enqueue_logs = true
+
   # Raises error for missing translations.
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 
-  # Use an evented file watcher to asynchronously detect changes in source code,
-  # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
-
-  config.hosts += [
-    "locations-api.dev.gov.uk",
-  ]
```

Comment:
We'll need to keep the bespoke host config for local docker compatibility, as Rails will block requests for locations-api.dev.gov.uk without.

Reply:
Ah, this always catches me out. I've done a bit of digging around and I think we do want to keep the config, but not in this form - we actually want config.hosts.clear for local dev:

https://github.com/alphagov/support-api/commit/2cbd2688bdad9284d6fd38fc5c39164071331623

Reply:
Done